### PR TITLE
feat: added static screens

### PR DESCRIPTION
**Pull request by Chakravarthi0**
preview: https://deploy-preview-1--wittest.netlify.app/

added the following components:
 - password input
 - signin card
 - signup card
 - quiz card
 - quiz quit modal
 - question
 
added the following pages:
 - home
 - category quiz
 - question 
 - rules
 - result
 - page not found
 - sign in
 - sign up

### CODE REVIEW

File: src/pages/question-page/question-page.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 1.8rem;
+    padding: 1.5rem;
+  }
+  .next-btn {
+    padding: 6px;
+  }
+  .quiz-stat > p {
+    font-size: 1.6rem;
+  }
+}
```
Comment (deekshithmd) [inline src/pages/question-page/question-page.css:50]: UI looks simple and clean, code looks  clean and good you can proceed.

**Review by deekshithmd — APPROVED** [review]

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,73 @@
...
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+:root {
+  --dark-purple: #49118c;
```
Comment (manojkumar8400) [inline src/index.css:2]: just a suggestion - you can use this type of variable name purple-900 instead of dark purple color.

**Review by manojkumar8400 — APPROVED** [review]

**Review by manojkumar8400 — COMMENTED** [review]
else code is readable, good to merge.

**Review by bipin7yadav — APPROVED** [review]
good to go

**Pull request merged**